weed/filer/redis2/universal_redis_store_test.go

```
@@ -0,0 +1,268 @@
+		Attr:     filer.Attr{Crtime: now, Mtime: now, Mode: 0644, TtlSec: ttlSec},
+	}); err != nil {
+		t.Fatalf("InsertEntry %s: %v", path, err)
+	}
+}
+
+func listNames(t *testing.T, store *UniversalRedis2Store, dir util.FullPath) []string {
+	t.Helper()
+
+	names := []string{}
+	if _, err := store.ListDirectoryEntries(context.Background(), dir, "", true, 100, func(entry *filer.Entry) (bool, error) {
+		_, name := entry.FullPath.DirAndName()
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "FullPath" from selector

(staticcheck)